### PR TITLE
ISSUE-134: SchemaRegistryClient should check empty java.auth.login.co…

### PR DESCRIPTION
**Pull request by harshach**
…nfig param.

### CODE REVIEW

File: schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java [hunk 1]
```diff
@@ -143,7 +143,7 @@
 
     static {
         String jaasConfigFile = System.getProperty("java.security.auth.login.config");
-        if (jaasConfigFile != null) {
+        if (jaasConfigFile != null && !jaasConfigFile.isEmpty()) {
```
Comment (satishd) [inline schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java]: You may want to say `!jaasConfigFile.trim().isEmpty()`

**Comment by satishd** [comment]
Closes #134

**Comment by priyank5485** [comment]
+1

**Comment by harshach** [comment]
@satishd done

**Comment by harshach** [comment]
and merge it in

**Pull request merged**